terminus-ui/cohort-date-range/src/cohort-date-range.component.html

```diff
@@ -0,0 +1,21 @@
+<div>
+  <ts-date-range
+    class="ts-cohort-date-range__date-range"
+    [dateFormGroup]="dateRangeFg"
+    [isDisabled]="!allowCustomDates || isDisabled || disableDateRange"
+    (dateRangeChange)="cohortDateRangeChange($event)"
+  ></ts-date-range>
+  <ts-select
+    class="ts-cohort-date-range__select"
```

Comment:
Seems like we're missing a label for the select. (noticed cause the top example in the PR image shows a select that is just blank)